packages/fx-core/tests/plugins/solution/solution.arm.test.ts

```diff
@@ -390,15 +334,155 @@ Mocked bot provision orchestration content. Module path: './provision/botProvisi
       `@secure()
 param provisionParameters object
 param provisionOutputs object
-Mocked frontend hosting configuration orchestration content. Module path: './teamsFx/frontendHostingConfig.bicep'.
-Mocked identity configuration orchestration content. Module path: './teamsFx/identityConfig.bicep'.
-Mocked aad configuration orchestration content. Module path: './teamsFx/aadConfig.bicep'.
 Mocked bot configuration orchestration content. Module path: './teamsFx/botConfig.bicep'.`.replace(
         /\r?\n/g,
         os.EOL
       )
     );
   });
+
+  it("add bot capability on tab app with simple auth success", async () => {
+    // Arrange
+    mockedCtx.projectSettings!.solutionSettings = {
+      hostType: HostTypeOptionAzure.id,
+      name: "azure",
+      activeResourcePlugins: [
+        aadPlugin.name,
+        fehostPlugin.name,
+        identityPlugin.name,
+        simpleAuthPlugin.name,
+      ],
+      capabilities: [TabOptionItem.id],
+    };
+    TestHelper.mockedFehostGenerateArmTemplates(mocker);
+    TestHelper.mockedAadGenerateArmTemplates(mocker);
+    TestHelper.mockedIdentityGenerateArmTemplates(mocker);
+    TestHelper.mockedSimpleAuthGenerateArmTemplates(mocker);
+    TestHelper.mockedBotGenerateArmTemplates(mocker);
+    TestHelper.mockedFeHostUpdateArmTemplates(mocker);
+    TestHelper.mockedIdentityUpdateArmTemplates(mocker);
+    TestHelper.mockedSimpleAuthUpdateArmTemplates(mocker);
+    // Action
+    let result = await generateArmTemplate(mockedCtx, [
+      aadPlugin,
+      botPlugin,
```

Comment:
the bot plugin should be fehostPlugin here?

Reply:
thanks!